zcash_client_sqlite/Cargo.toml

```diff
@@ -14,6 +14,7 @@ edition = "2021"
 rust-version = "1.60"
 
 [dependencies]
+incrementalmerkletree = { version = "0.3", features = ["legacy-api"] }
```

Comment:
```suggestion
```

Reply:
This is still present, and AFAICT is not used?

Reply:
Removing this would be pointless because it will be immediately reintroduced for the `Position`, `Address` and other types with the shardtree integration.